people/johannesschrott/.htaccess

```
@@ -0,0 +1,7 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Rewrite engine setup
+RewriteEngine On
+
+Redirect 302 / https://www.jku.at/en/institute-for-application-oriented-knowledge-processing/about-us/team/johannes-schrott/
```

`Redirect` requires full URL, `RewriteRule` is relative.  Usually easier to use this pattern:
```suggestion
RewriteRule ^$ https://www.jku.at/en/institute-for-application-oriented-knowledge-processing/about-us/team/johannes-schrott/ [R=302,L]
```

---

Thank you for suggesting a fix for this!